### PR TITLE
Hannah Native App

### PR DESCRIPTION
**Pull request by jesinkey**
My reflections on how this week's project turned out:
Im happy to having a "working" app with navigation and an API call. Im not done with styling however and will continue to complete this next week after the collab days. 

Things I'd like to have clarified or explained in more detail:
I thought this entire week was a little vague and would have like to have more guidance as we had to learn: native, expo, new style of debugging and new styling all in one week. Maybe split this over two weeks instead?

### CODE REVIEW

File: components/CardThumb.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  /*Deconstructing props*/
+  const { card, navigation } = props;
+
+  /*Function to create route for CardDetails when clicking the entire CardThumb*/
+  const onPressCardButton = () => {
+    navigation.navigate('CardDetails', { card: card });
+  };
+
+  return (
+    <>
```
Comment (KarolCislowski) [inline components/CardThumb.js:30]: you are returning a single component here, so there is no need to using react fragment.

**Review by KarolCislowski — CHANGES_REQUESTED** [review]
Please add the link to the hosted app for testing.

File: components/CardThumb.js (hunk 1 above)
Comment (jesinkey) [reply components/CardThumb.js:30]: Done!

**Comment by jesinkey** [comment]
Hi! I have now added the published project in README

**Review by KarolCislowski — APPROVED** [review]
Great Project 😸

**Pull request closed without merging**